### ISSUE TITLE
When can new LC release be expected?

### ISSUE THREAD

**Issue by Ralkniv** [t-d-k/LibreCrypt#52]
Quite a lot of time has passed since the latest LibreCrypt update. Even though not all suggestions and reported bugs have been sorted out by now, would it be possible to expect a release of at least what has been successfully done so far?

**Comment by ssdnvv** [comment]
Personally I assume the project is dead - the developer hasn't answered in quite a long time, no matter whatever question has been asked... what is really sad :(

**Comment by t-d-k** [comment]
I don't think there have been enough changes to justify a new release yet. There will be a release if I fix any major bugs, or if I get a code signing certificate. 
Currently the donations are £70  (thanks everyone!)  which is about halfway there.

**Comment by ssdnvv** [comment]
Ok, how much do you still need - another £70?

**Comment by t-d-k** [comment]
Yes, roughly: https://www.digicert.com/code-signing/kernel-mode-certificates.htm

**Comment by ssdnvv** [comment]
And with that £70 you'll be able to sign the executables so we won't have to switch to testmode under 64bit-Windows anymore (respectively I would be able to run the portable version on machines where I have local admin rights yet not the right to switch to testmode)?

**Comment by crylium** [comment]
I've just donated £10 - hope this helps, at least a little bit.

**Comment by ssdnvv** [comment]
If I manage to correctly open my partition, I will give the missing part for the cert (if t-d-k confirms my questions)

**Comment by t-d-k** [comment]
@ssdnvv Yes, to your questions.
@crylium Thanks! Don't forget to let me know if you want to be credited.

**Comment by crylium** [comment]
@t-d-k Wasn't even aware of that. If you can mention my GitHub username then great, not a biggie otherwise. I still use FreeOTFE v5.21 as I couldn't get LibreCrypt to open my LUKS volume. Looking forward for a new LibreCrypt release therefore.

**Comment by Ralkniv** [comment]
t-d-k pls make sure before releasing upgrade that drag-and-drop to enter the password to the edit box is enabled as it is not the case in the latest (6.2.5806.4070) EXE build. It would be a pity and the prog would be prone to easy keylogging and clibboardlogging schemes. A small detail that makes a huge difference...

**Comment by t-d-k** [comment]
@Ralkniv Thanks for reporting this. I have moved your comment to a new issue #54 , just to keep things organised.

**Comment by NHellFire** [comment]
Open source projects can get a cheap certificate from [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml).  However, Windows 10 [requires EV certs](https://blogs.msdn.microsoft.com/windows_hardware_certification/2016/07/26/driver-signing-changes-in-windows-10-version-1607/) for drivers.

**Comment by Ralkniv** [comment]
14 months have passed since I opened this thread enquiring about a possible update - at least containing those issues that have been fixed. Would the developer possibly consider uploading such an update? Many of us would be most grateful.

**Comment by crylium** [comment]
@Ralkniv I suspect the project isn't maintained anymore...
